spec/helpers/controllers_spec_helper.rb

```diff
@@ -0,0 +1,19 @@
+module ControllerHelpers
+  def login_admin
+    before(:each) do
```

Comment:
I wouldn't include the `before(:each)` in this method, since we don't know how we will use it in tests.
What about if I want to use it as a global signin, like in a `before(:all)`?

This method provides a way to login as an admin, and it's good to do so. It shouldn't make any assumption about the way we have to use it, and let the developper choose the best way. :)